styles.css

```
@@ -1 +1,82 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #FEFAE0;
+}
+
+.container {
+    margin: 20px;
+    background-color: #FEFAE0;
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-template-rows: 50px auto 50px 50px 50px 50px 50px;
```

@abouman76  - de oplossing is goed, het is nog beter om de template-rows als volgt te omschrijven:
```suggestion
    grid-template-rows: 50px auto repeat(5, 50px);
```